### ISSUE TITLE
Larger Teleporter Frames

### ISSUE THREAD

**Issue by koalalord9999** [mekanism/Mekanism#7001]
**Describe the the feature you'd like**
I just stumbled upon a weird coinquidink... and thought, man... it would be GREAT if I could make a larger teleporter frame... (would make teleporting tall mobs, such as endermen and withers possible.

**Describe alternatives you've considered**
Draconic evolution teleporters come to mind, as does immersive portals, but I don't have either of those in my pack, and I'm on a server that wont add new mods.

**Additional context**
Basically, I just want to be able to make a larger teleporter frame, similar to how Mojang added almost infinite size nether portals a long time ago.
![image](https://user-images.githubusercontent.com/9123026/108147806-31a0ec80-7095-11eb-8a4e-ba95b70bcc7a.png)
![image](https://user-images.githubusercontent.com/9123026/108147887-539a6f00-7095-11eb-8959-00f3ea87ce38.png)
